src/Commands/tfchill.js

```diff
@@ -113,7 +113,7 @@ export default class extends Command {
             };
 
             // adds createReactionCollector to the embed we sent, so we can 'collect' all the correct answers
-            const collector = msgEmbed.createReactionCollector(filter, { time }); // will only collect for n seconds
+            const collector = msgEmbed.createReactionCollector(filter, { max: 1, time }); // will only collect for n seconds
```

Comment:
The max cannot be set here because the purpose of chill mode is that all users have the opportunity to answer within the time limit. Also, changes will need to be applied to both `tfchill` and `mcchill` game modes.